### ISSUE TITLE
Hi I got LambdaLR` doesn't follow PyTorch's LRScheduler API while trying to run LambdaLR` doesn't follow PyTorch's LRScheduler API

### ISSUE THREAD

**Issue by kentdan** [facebookresearch/hyperreel#25]
Error executing job with overrides:
['experiment/dataset=immersive', 'experiment/training=immersive_tensorf', 'experiment.training.val_every=5',
'experiment.training.test_every=10', 'experiment.training.ckpt_every=10', 'experiment.training.render_every=30',
'experiment.training.num_epochs=30', 'experiment/model=immersive_sphere', 'experiment.params.print_loss=True',
'experiment.dataset.collection=02_Flames', '+experiment/regularizers/tensorf=tv_4000',
'experiment.dataset.start_frame=0', 'experiment.params.name=immersive_02_Flames_start_0']

pytorch_lightning.utilities.exceptions.MisconfigurationException: The provided lr scheduler `LambdaLR` doesn't follow PyTorch's LRScheduler API. You should override the `LightningModule.lr_scheduler_step` hook with your own logic if you are using a custom LR schedule

**Comment by benattal** [comment]
Did you manage to resolve the problem?

**Comment by jmpageau** [comment]
I am alose interested i got the same error
